docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,276 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #52.        chunk_stop = query_idx
</pre><p>This is based on the original algorithm proposed in the paper, where exclusion zone is set to <code>m</code>. This will be changed later to reflect the default <code>excl zone</code> in STUMPY.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>As explained by <a href="https://github.com/seanlaw" target="_blank">@seanlaw</a> in PR#872, we should consider <code>excl_zone</code>s so that the output from DAMP to be the same as the discord discovered via <code>stumpy.stump</code>  </p><p> </p><p>Since the naive DAMP uses <code>STUMPY.config.STUMPY_EXCL_ZONE_DENOM</code> , I think it is better to add the <code>excl_zone</code> now rather than later.  </p>